packages/story-editor/src/components/panels/design/textStyle/stylePresets.js

```diff
@@ -82,6 +81,7 @@ function PresetPanel({ pushUpdate }) {
   const [isPopupOpen, setIsPopupOpen] = useState(false);
   const { isRTL, styleConstants: { topOffset } = {} } = useConfig();
   const hasPresets = textStyles.length > 0;
+  const SPACING = { x: isRTL ? 12 : 46 };
```

Comment:
Does the popup here need `isRTL` and the difference in spacing?

Also - can this live in the constants file with the other spacing constants?

Reply:
Yes, it is docked to the `inspector` element. Since docking is currently based on the left of the element (in conjunction with the where the anchor is), when in RTL that spacing only needs to give the appearance of a little bit of margin, hence the 12 vs 46. I think it is fine to leave here since the other spacing constants are related to the color picker